package.json

```diff
@@ -33,12 +33,12 @@
     "yarn": ">= 1.3.0"
   },
   "scripts": {
-    "ci": "yarn run eslint && yarn run test:coverage",
+    "ci": "yarn run eslint && yarn run test && yarn run test:coverage",
     "clean": "rm -Rf tmp",
     "mongo": "mongod --dbpath data --quiet",
     "mongo:win": "mongod --dbpath /data/db --quiet",
-    "test": "yarn run mocha && yarn run cucumber",
-    "test:coverage": "nyc --reporter=lcovonly --reporter=text-summary yarn run test",
+    "test": "nyc --reporter=text-lcov yarn run mocha && yarn run cucumber",
```

Comment:
Does `nyc` consider the coverage by both mocha and cucumber? Do we want to have code coverage for `test` at all?

Reply:
Im not sure to be honest. I would say we have to keep a look on it.